src/components/Header.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { NavBar } from "./Navbar";
```

Comment:
This line is throwing an error. Please make sure to push up working code before creating a pr